src/components/ProductTour/ProductTour.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import styles from './ProductTour.scss';
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+    /** Should the tour be shown */
+    isOpen: boolean;
+    /** An array of ReactElement where each element is one slide */
+    children: React.ReactElement[];
```

Comment:
Would it be an idea to constrain it to be of HTML type `section`, so that it will throw a warning when using something that isn't suitable (like `span` or whatever - now you could in theory use `button` as a slide container)

Reply:
I'm not sure how useful that is, and how easy is it to implement. E.g. how to validate when the slide is an other React Element, e.g. `MySlide` that in turn can return different top elements, or even a fragment?

Reply:
I see. Then perhaps it might be a good idea to include some `<Slide />` component ourselves, because then we're sure a correct element will be used as wrapper and we retain full control over how that content wrapper is to be styled within the context of OneUI rather than the application.
For example, say we decide to restyle this component some time in the future, and the content needs more padding. Then we simply change that in OneUI without having to adapt applications that use this component.

Reply:
I was thinking to just wrap the children, whomever they are in `<section>` and add styling there. Do you think that would work?

Reply:
Yes, that should indeed work too. I was just thinking if this API would be a bit too flexible but maybe it's fine to keep it simple.

Reply:
Could you add some spacing sizes to the ticket description? Both for the content, and maybe for the dots as well. Thanks

Reply:
Slides have a padding of 25px all around (you don't necessarily need to use OneUI spacing variables in this case)
Dots are positioned in the middle / center of the footer, sizes are already described in the ticket: 10x10px
The footer in the design is 60px high (padding around the button is 15px)